### PR TITLE
Update CUDA to version 7.5.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
please test

**Comment by ghost** [comment]
please test

File: cuda.spec [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
 
-Source: http://developer.download.nvidia.com/compute/cuda/7_0/Prod/local_installers/%{n}_%{realversion}_linux.run
+Source: http://developer.download.nvidia.com/compute/cuda/7.5/Prod/local_installers/%{n}_%{realversion}_linux.run
+
+Provides: libcuda.so.1
+Provides: libgcc_s.so.1
+Provides: libgcc_s.so.1(GCC_3.0)
+Provides: libgcc_s.so.1(GLIBC_2.0)
+Provides: libstdc++.so.6
+Provides: libstdc++.so.6(GLIBCXX_3.4)
```
Comment (davidlt) [inline cuda.spec]: Replace those with: `AutoReqProv: no`

**Comment by ghost** [comment]
@davidlt thanks for the tip, updating now.

**Comment by ghost** [comment]
please test

**Pull request merged**